discord/index.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import discord 
+from discord.ext import commands, tasks
+import aiohttp 
+from bs4 import BeautifulSoup
+import config
+
+called = False
+bot=commands.Bot(command_prefix=commands.when_mentioned_or(config.PREFIX), 
+                intents = discord.Intents.default()
+)
+
+@bot.event
+async def on_ready():
+    await bot.change_presence(activity=discord.Game(name='Watching Scirate'))
+    print(f'Logged in as {bot.user.name}')
+    return
+
+@bot.command()
+async def top(ctx):
```

Comment:
Over on the issue - https://github.com/scirate/scirate/issues/430 - aram mentions that it would be good that this runs at the right time, so that people get a chance to put their scites in before it queries them.

A good time would be, I think, 4PM GMT; is it possible to put this in the schedule somehow?

Reply:
Are you on discord by any chance? I could show you how this works and you can tell me what exactly you are looking for